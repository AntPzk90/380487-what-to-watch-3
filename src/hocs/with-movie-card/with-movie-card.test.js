import React from 'react';
import renderer from 'react-test-renderer';
import withMovieCard from './with-movie-card.jsx';

const isPlayMock = false;

it(`SnapshotTest withMovieCard`, () => {

  const tree = renderer
  .create(
      <withMovieCard
        isPlay = {isPlayMock}
        onCardMouseHove = {() => {}}
        onCardMouseLeave = {() => {}}
      />
  )
  .toJSON();

  expect(tree).toMatchSnapshot();
});
