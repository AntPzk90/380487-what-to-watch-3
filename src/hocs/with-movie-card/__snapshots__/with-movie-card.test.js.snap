// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SnapshotTest withMovieCard 1`] = `
<withMovieCard
  isPlay={false}
  onCardMouseHove={[Function]}
  onCardMouseLeave={[Function]}
/>
`;
